Documentation/Development.md

```diff
@@ -38,25 +38,25 @@ $ swift build
 
 ### Linux
 
+Install the following dependencies of SourceKit-LSP:
+
+* libsqlite3-dev, libncurses5-dev, python, ninja-build
+
 ```sh
 $ export PATH="<path_to_swift_toolchain>/usr/bin:${PATH}"
 $ swift package update
 $ swift build -Xcxx -I<path_to_swift_toolchain>/usr/lib/swift -Xcxx -I<path_to_swift_toolchain>/usr/lib/swift/Block
 ```
 
+Setting `PATH` as described above is important even if `<path_to_swift_toolchain>/usr/bin` is already in your `PATH` because `/usr/bin` must be the **first** path to search.
```

Comment:
> because `/usr/bin` must be the **first** path to search.

Why would it need to be first? Obviously if you have more than one toolchain on your path you need the right one to come before the wrong one, but that's just normal PATH behaviour.

Reply:
The problem is that in the docker container, we have the following `PATH`: 
```
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

`sourcekit-lsp` finds `libsourcekitdInProc` and `libIndexStore` relative to `/usr/sbin` in `/usr/sbin/../lib/` (aka. `/usr/lib`). Thus `/usr/sbin` is not discarded and it becomes the default toolchain. But it doesn’t contain the swift (or clang) executable, causing all sorts of later failures. If `/usr/bin` is before `/usr/sbin` in `PATH`, we don’t hit this issue.

Reply:
Okay, the note is fine for now, but this seems like something we should fix.  We could recognize when one toolchain is a strict superset of another one (sbin and bin have the same libraries but only bin has the executables).